tests/unit/s2n_build_test.c

```diff
@@ -82,24 +83,24 @@ S2N_RESULT s2n_check_supported_libcrypto(const char *s2n_libcrypto)
         { .libcrypto = "openssl-1.0.2", .is_openssl = true },
         { .libcrypto = "openssl-1.1.1", .is_openssl = true },
         { .libcrypto = "openssl-3.0", .is_openssl = true },
+        { .libcrypto = "openssl-3.0-fips", .is_openssl = true, .is_opensslfips = true },
```

Comment:
Is `is_opensslfips` set to false anywhere?

Reply:
When initializing a struct this way in C, any field not specified is "0", which is false. So everywhere that doesn't set is_openssl_fips gets "false".